src/test/java/SwerveTest.java

```diff
@@ -2,38 +2,67 @@
 
 import java.util.Arrays;
 
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
+import edu.wpi.first.hal.HAL;
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.kinematics.ChassisSpeeds;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.simulation.DriverStationSim;
+import edu.wpi.first.wpilibj.simulation.SimHooks;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.commands.DriveDistanceCommand;
+import frc.robot.commands.DriveTimeCommand;
 import frc.robot.subsystems.DriveSubsystem;
 
 public class SwerveTest {
```

Comment:
Is this whole file just to save you some time when you were doing repeated testing of the swerve modules? Should we keep this in when we merge into main?

Reply:
Unit tests are important. This will be expanded later on.